content/en/apps/reference/app-settings/transitions.md

```diff
@@ -671,4 +672,63 @@ If this configuration is not set then the message defaults to what is set in the
     ]
   }
 ]
-```
\ No newline at end of file
+```
+
+### create_user_for_contacts
+
+An existing offline user can be replaced on a device so that a new user can use that device without needing to immediately sync with the server. _Available since 3.17.x._
```

Comment:
```suggestion
An existing offline user can be replaced on a device so that a new user can use that device without needing to immediately sync with the server. _Available since 4.1.x._
```